docs/references/authentication_oidc.md

```diff
@@ -45,3 +45,32 @@ secret/ruser-adminepinioio-9341763ee7dcbce070e7c14f246ec8291e9a7278 patched
 
 You're now able to login with the credentials in Epinio UI.
 
+## Groups and Roles mapping
+
+The external identity providers can sometimes provide additional information about the user, for example the groups that he's member of.  
+
+These groups can be used to associate a specific role to the user. To do so you need to add a `rolesMapping` key to the `dex-config` secret. The value of the key is a yaml string that will be used to map the groups of a provider to a specific role:
+
+```
+rolesMapping: |-
+  - connectorId: github
+    groups:
+    - id: Org1:Admins
+      role: admin
+    - id: Org1:TeamBlue
+      role: user
+
+config.yaml: |-
```

Comment:
What is the purpose of the `config.yaml` key ? It is not described in the preceding text, nor after.

Reply:
It's the Dex configuration itself. At the beginning it was the only key in this secret, so it was straightforward. The we started adding more keys related on how Epinio integrates with Dex. Should I add something at the beginning maybe?

Line 23: https://github.com/epinio/docs/pull/191/files#diff-88a112af3f97d3ebffec4f38eede241145a6b74d7495746edec9caf36beae58dR23

Reply:
While the referenced line, i.e.

> To add more connectors, you can edit the Dex config file deployed with the `dex-config` secret

mentions the config file it is not fully clear that this "file" is the `config.yaml` key in the secret.
I would make that link a bit clearer.

Reply:
And when I said "preceding text" I only had looked at the text in the diff, not the entire file. That was my fault.